src/constraints.jl

```diff
@@ -164,13 +162,13 @@ Return a `Bool` indicating whether the set of type `S1` in constraint `c` can be
 If `c` is a `ConstraintIndex{ScalarAffineFunction{Float64},LessThan{Float64}}`,
 
 ```julia
-cantransformconstraint(model, c, GreaterThan(0.0)) # true
-cantransformconstraint(model, c, ZeroOne())        # false
+cantransform(model, c, GreaterThan(0.0)) # true
+cantransform(model, c, ZeroOne())        # false
```

Comment:
This is false for all current solvers, but MOI allows you to implement a solver where this could work. Maybe clarify that this is expected behavior but not a hard rule.